GitFyle.Core.Api/Brokers/Storages/StorageBroker.ContributionTypes.References.cs

```diff
@@ -0,0 +1,26 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using GitFyle.Core.Api.Models.Foundations.ContributionTypes;
+using Microsoft.EntityFrameworkCore;
+
+namespace GitFyle.Core.Api.Brokers.Storages
+{
+    internal partial class StorageBroker
+    {
+        void SetContributionTypeReferences(ModelBuilder modelBuilder)
+        {
+            int maxStringLength = 255;
+
+            modelBuilder.Entity<ContributionType>()
+                .Property(contributionType => contributionType.Name)
+                .HasMaxLength(maxStringLength)
+                .IsRequired();
+
+            modelBuilder.Entity<ContributionType>()
+                .Property(contributionType => contributionType.Value)
+                .IsRequired();
+        }
```

Comment:
Add the unique constraint on Name to ensure we do not get duplicate contribution types.
```cs
            modelBuilder.Entity<ContributionType>()
                .HasIndex(source => source.Name)
                .IsUnique();
```